code/modules/events/immovable_rod.dm

```diff
@@ -67,6 +67,8 @@ In my current plan for it, 'solid' will be defined as anything with density == 1
 	var/num_sentient_mobs_hit = 0
 	///How many people we've hit with clients
 	var/num_sentient_people_hit = 0
+	/// The rod levels up with each kill, increasing in size and auto-renaming itself.
+	var/dnd = TRUE
```

Comment:
cute variable name, but unreadable, please name it something useful 🙂

Reply:
![image](https://user-images.githubusercontent.com/24975989/107441585-61983f00-6b2d-11eb-95ca-4d0e24d855ac.png)